features/steps/steps.go

```diff
@@ -335,3 +333,15 @@ func (c *Component) thereIsATopicAPIThatReturnsTheRootTopicAndTheSubtopicForRequ
 
 	return nil
 }
+
+func (c *Component) getPageDataRequestToZebedeeIsSuccessful() error {
+	// The page type needs to be either an article, a bulletin, or a compendium
+	c.FakeAPIRouter.setJSONResponseForGetPageData(200)
+	return nil
+}
+
+func (c *Component) getPageDataRequestToZebedeeFindsAWrongPageType() error {
+	// The page type needs to be either an article, a bulletin, or a compendium, otherwise a 404 is returned
+	c.FakeAPIRouter.setJSONResponseForGetPageData(404)
+	return nil
+}
```

Comment:
This isn't finding the wrong page type, it's getting a 404 from zebedee instead which is slightly different.

Reply:
I've changed the setJSONResponseForGetPageData function to get a wrong page type now.